css/analyzer.css

```diff
@@ -171,8 +171,9 @@
     color: #f00 !important;
 }
 
+.snippet-editor__form .snippet-editor__field--invalid,
```

Comment:
We shouldn't have to do this, there is probably an underlying problem we should instead solve. The point of BEM is reducing the specificity of the CSS selectors.

Please find the source of CSS that makes you write this line and solve it in a better way.